.env

```diff
@@ -0,0 +1,2 @@
+PORT=4000
+HOST=http://localhost:
```

Comment:
What do you think about having just `http://localhost` without `:` for port?
You could also create a util for getting these two properties e.g. `getServerURL()` or something like that and use it in `seed/avatars.ts`.

Reply:
There should be a port as React/Angular or Vue also runs on localhost.